src/changelog/2024/12/device-editor-cache.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Device Editor Access Speed Up
+description:
+date: 2024-12-18 14:00:00.0
```

Comment:
Since release 2.1.2 is called out, I think the date should be set for tomorrow

```suggestion
date: 2024-12-19 14:00:00.0
```

Reply:
It's live on Production now, so this could be published today, but no problem.

Reply:
either or Ben. I tend not to mention the release unless the changelog is being published on the day of release (to avoid confusion for cloud users)

I will approve and merge if you wish.